Framework/DataHandling/src/LoadIsawDetCal.cpp

```diff
@@ -210,15 +210,16 @@ void LoadIsawDetCal::exec() {
       }
     }
     boost::shared_ptr<RectangularDetector> det;
-    std::ostringstream Detbank;
-    Detbank << "bank" << id;
+    std::string Detbank = "bank" + std::to_string(id);
```

Comment:
It does look suspicious. I created a free function called `getBankName` that uses the logic from a bit further down, but am not 100% sure this is correct.